src/content/docs/configuration/targets.mdx

```diff
@@ -0,0 +1,329 @@
+    MyTarget    docker-provider     {
+                                        "Sock Path": "/var/run/docker.sock"
+                                    }
+   ```
+
+## Delete a Target
+
+Daytona allows you to delete Targets, helping you manage and remove those that are no longer needed. Once a Target is deleted, you will not be able to manage or deploy Workspaces on that Target.
+
+1. Run the following command:
+
+    ```shell
+    daytona target delete
+    ```
+
+    Upon running this command, Daytona will display a list of your Targets with their details. You will be able to see the Target name, the unique identifier of the Workspace, and the repository it is connected to.
+
+2. Press `Enter` on the selected Target to delete it.
+
+    ```text
+        Choose a Target
+        1 item
+        ===
+        MyTarget          
+        docker-provider
+    ```
+
+3. Confirm the action.
+
+    ```text
+        Delete all workspaces within MyTarget?
+        You might not be able to easily remove these workspaces later.
+        
+        [Yes]     [No]
+    ```
+
+    ```text
+        Target MyTarget removed successfully
+    ```
+
+## Docker
+
+Daytona allows you to set a Docker Target to use for deploying and managing Workspaces. The Docker Target enables you to deploy Workspaces on your local or remote machine using Docker.
+
+1. Run the following command to set a Target:
+
+    ```shell
+    daytona target set
+    ```
+
+2. Select `docker-provider` to use the Docker Provider.
+
+    ```text
+        Choose a provider
+        1 item
+        ===
+        docker-provider
+        v0.0.0
+    ```
+
+3. Select `local` to set a local Docker target, or `New Target` to set a remote Docker target.
+
+    - A local machine uses your local installation of Docker to deploy Workspaces and requires configuring the `Sock Path` to the Docker socket.
+
+    - A remote machine uses a remote installation of Docker to deploy Workspaces and requires configuring the remote `password`, `port`, `user`, `sock path`, `Workspace data directory`, and `private key path`.
+
+4. Enter a name for your Target.
+5. Enter the appropriate configuration options when prompted. The configuration options vary based on the selected Provider. The following example shows setting a local Docker Target.
+
+    ```text
+        Sock Path
+        > /var/run/docker.sock
+    ```
+
+6. Click `Enter` to confirm setting the Target.
+
+    ```text
+        Target set successfully
+     ```
+
+## AWS
+
+Daytona allows you to set an AWS Target to use for deploying and managing Workspaces.
+
+1. Run the following command to set a Target:
+
+    ```shell
+    daytona target set
+    ```
+
+2. Select `aws-provider` to use the AWS Provider.
+
+    ```text
+        Choose a provider
+        1 item
+        ===
+        aws-provider
+        v0.0.0
+    ```
+
+3. Select `New Target`.
+4. Enter a name for your Target.
+5. Enter the appropriate configuration options when prompted.
+
+    ```text
```

Comment:
https://github.com/daytonaio/daytona-provider-aws/issues/3

The target option descriptions will probably be updated before the release so just putting it here for tracking